RELEASES.md

```diff
@@ -0,0 +1,87 @@
+# Releases
+
+## Versioning
+
+SOCI versions follows the `Major.Minor.Patch-PreRelease` scheme of [Semantic Versioning](https://semver.org).
+
+### `soci CLI / soci-snapshotter / SOCI library`
+
+The soci CLI, soci-snapshotter, and library all share the same version to make it easy to understand compatibility.
+
+### `ztoc data format`
+
+zTOC has it’s own version which will evolve independently from the other components of SOCI. A bump in zTOC version will translate to a bump in the SOCI version, but the reverse is not true. Since zTOCs are stored in content-addressable registries, changes to the ztoc format will be strongly discouraged to avoid churn for SOCI users.
+
+> Note: The zTOC data format is separately versioned, but the library that's used to build, interact with, and manipulate zTOCs is part of the SOCI library and follows the CLI/snapshotter version. 
```

Comment:
`> ` is generally reserved for [block]quotes.

Reply:
Fair. I've switched to this format:

*__NOTE:__ this is a note*